qa/L0_results/check_results.py

```diff
@@ -18,67 +18,94 @@
 import sys
 
 
-def add_arguments(parser):
-    parser.add_argument('-f',
-                        '--config-file',
-                        type=str,
-                        required=True,
-                        help='The config file for this test')
-    parser.add_argument('-d',
-                        '--export-path',
-                        type=str,
-                        required=True,
-                        help='The full path to the export directory')
+class TestOutputValidator:
```

Comment:
Can't comment on the file itself. I think  qa/L0_results/1.ckpt  was mistakenly added.

Reply:
This was not by accident. Some of the tests now contain checkpoints for speed. They do not need to profile, as they test only on `analyze` or `report` subcommands.